scripts/set-arm-package-url.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+const ROOT = path.join(__dirname, "..");
+const DEPLOY_DIR = path.join(ROOT, "deploy");
+const AZURE_DEPLOY = path.join(DEPLOY_DIR, "azuredeploy.json");
+const AZURE_DEPLOY_EXISTING = path.join(
+  DEPLOY_DIR,
+  "azuredeployexistingresource.json"
+);
+const AZURE_DEPLOY_EDITABLE = path.join(DEPLOY_DIR, "editableazuredeploy.json");
+const gitTag = process.argv[2];
+
+function setPackageUrl(armTemplate) {
+  const updatedPackageUrl = `https://github.com/Azure-Samples/communication-services-virtual-visits-js/releases/download/${gitTag}/sample.zip`;
```

Comment:
The `gitTag` needs to be URL encoded (using `encodeURIComponent()` method): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

For example, if the tag is "Release/1.0.5" the correct URL is:
https://github.com/Azure-Samples/communication-services-virtual-visits-js/releases/download/Release%2F1.0.5/sample.zip

Reply:
We'll need to encode only the gitTag and not the entire URL:
![2022-05-31 13_57_43-JSFiddle - Code Playground and 10 more pages - Work - Microsoft​ Edge](https://user-images.githubusercontent.com/43504546/171283419-cba645a4-402d-43c0-ad68-81e324915315.png)